lib/puppet/face/help.rb

```diff
@@ -146,7 +146,9 @@ def all_application_summaries()
       if (is_face_app?(appname))
         begin
           face = Puppet::Face[appname, :current]
-          result << [appname, face.summary]
+          # Add deprecation message to summary if the face is deprecated
+          summary = face.respond_to?(:deprecated?) && face.deprecated? ? face.summary + _(" (Deprecated)") : face.summary
```

Comment:
I think we ought to be able to drop it - any external faces should still be built on top of the `face_base` from the current puppet version.

Maybe if an updated version of the `help` face got pluginsync-ed to on old agent things would break?